src/internal/grid/grid.tsx

```diff
@@ -38,7 +38,11 @@ export default function Grid({ layout, children: render, columns }: GridProps) {
     return colspan * cellWidth + (colspan - 1) * gridGap;
   };
   const getHeight = (rowspan: number) => rowspan * rowspanHeight + (rowspan - 1) * gridGap;
-  const getColOffset = (x: number) => getWidth(x) + gridGap;
+  const getColOffset = (x: number) => {
+    const offset = getWidth(x) + gridGap;
+    const isRtl = document.documentElement.dir === "rtl";
```

Comment:
Updating to use the utility function and gridRef. Same with the previous comment we should probably do this in one place.